### DISCUSSION TITLE
lernos-zettelkasten/de/2-0-PKM-Grundlagen/

### DISCUSSION THREAD

**Discussion by giscus** [cogneon/lernos-zettelkasten#15] [Announcements]
# lernos-zettelkasten/de/2-0-PKM-Grundlagen/

https://cogneon.github.io/lernos-zettelkasten/de/2-0-PKM-Grundlagen/

**Comment by bluedecke** [comment]
Der Gliederungspunkt oben links (PKM Grundlagen) ist nicht identisch mit der Überschrift (Grundlagen der Zettelkasten-Methode). PKM ist nicht dasselbe wie die Zettelkasten-Methode.

**Reply by saertna** [reply]
Danke @bluedecke für den Hinweis. Da hast du vollkommen recht.
